universe/auto_syncer.go

```diff
@@ -846,18 +846,30 @@ func (f *FederationEnvoy) SyncAssetInfo(ctx context.Context,
 		// Sync failures are expected from Universe servers that do not
 		// have a relevant universe root.
 		if err != nil {
-			log.Debugf("asset lookup for %v failed with remote"+
-				"server: %v", assetID.String(), addr.HostStr())
-			//lint:ignore nilerr failure is expected and logged
+			log.Warnf("Asset lookup for %v failed with remote"+
+				"server '%v': %v", assetID.String(),
+				addr.HostStr(), err)
+
+			// We don't want to abort syncing here, as this might
+			// just be one server in our list and returning an error
+			// would cause us to stop trying the other servers.
+			// lint:ignore nilerr failure is expected and logged.
 			return nil
 		}
 
 		// There should only be one sync diff since we're only syncing
 		// one universe root.
 		if syncDiff != nil {
 			if len(syncDiff) != 1 {
-				log.Debugf("unexpected number of sync diffs: "+
-					"%v", len(syncDiff))
+				log.Warnf("Unexpected number of sync diffs %d "+
+					"when looking up asset %v on remote "+
+					"server '%v': %v", len(syncDiff),
+					assetID.String(), addr.HostStr(), err)
```

Comment:
Here `err` is always `nil`, because non-nil err would cause `return` in the branching at line 857.